spec/controllers/policies_controller_spec.rb

```diff
@@ -1,30 +1,94 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
-
 require 'parallel'
+
 DatabaseCleaner.strategy = :truncation
 
-describe PoliciesController, type: :controller do
+describe PoliciesController, type: :request do
+
+  before(:all) do
+    # there doesn't seem to be a sane way to get this
+    @original_database_cleaner_strategy =
+      DatabaseCleaner.connections.first.strategy
+        .class.name.downcase[/[^:]+$/].intern
+
+    # we need truncation here because the tests span many transactions
+    DatabaseCleaner.strategy = :truncation
+
+    # init Slosilo key
+    Slosilo["authn:rspec"] ||= Slosilo::Key.new
+  end
+
+  after(:all) { DatabaseCleaner.strategy = @original_database_cleaner_strategy }
```

Comment:
There's really no reason to inline this statement

Reply:
Fits on one line is a good reason for me.  I don't mind it changing it though.